### ISSUE TITLE
[Bug] Site list has become un-editable

### ISSUE THREAD

**Issue by Coyote21** [darkreader/darkreader#11615]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

I found a work around (below) so this bug report might only serve as notice or for others experiencing a similar issue.

I'm using the latest version of Brave browser [Version 1.57.49 Chromium: 116.0.5845.96 (Official Build) (64-bit)]
and latest version of Dark reader extension [Version 4.9.65]

My site list has become un-editable. I can type a new URL in the textbox at the bottom but nothing happens when I hit enter, If I click out of the URL textbox the text remains but if leave the Site List tab (e.g switch to Filter) and then return the new URL is not saved.

I'm also unable to edit or remove existing sites, again I can edit or delete the text but there is no way to save the changes.

I don't know when this started happening for sure b/c I don't add new sites very often.

It's only an issue on my current profile, a new profile with the Dark Reader extensions works fine (Sorry, had to lie in the Prereq so I could post). So I decided to research how to export/import my site list so I could just start again with a new profile. During my research I discovered the 'Preview New Design' (Which looks great BTW) in the Dev Tools.

When using the new design everything works fine. I can add, remove, edit all the URLs on my site list. However, when switching back to the old design the issue returns.

TLDR; THE SOLUTION:
Switch to the 'New Design' - Button available in Dev Tools. 

### Website Address

any

### Steps To Reproduce

Don't know what I did to create the issue, but to reproduce it I just try to add a new URL to the site list or edit an existing one.

### Expected Behavior

I expect my changes to be saved after I leave the Extension configuration window.

### Actual Behavior

My Changes are NOT saved

### Operating System

Windows 10

### Web Browser name and version

Brave browser [Version 1.57.49 Chromium: 116.0.5845.96 (Official Build) (64-bit)]

### Dark Reader version

Version 4.9.65

### Additional Context

Sorry again for lying in the prerequisite section....

**Comment by daniteq** [comment]
I was having the same issue on Edge browser. As OP mentioned, I switched to new design and I was able to add sites to list again

**Comment by yeagerghost** [comment]
Same issue on Chromium.  Switched to new design as suggested and was then able to add sites to the list

**Comment by donfiguerres** [comment]
@yeagerghost Do you have the steps for the workaround? Could you post it here please? I'm having the same issue.

**Comment by o-l-a-v** [comment]
@donfiguerres 

First:

<img width="219" alt="image" src="https://github.com/darkreader/darkreader/assets/6450056/bac4aa82-c2a0-48e0-8ee8-f495db16cfb2">

Then:

<img width="439" alt="image" src="https://github.com/darkreader/darkreader/assets/6450056/0adc962b-25fd-476e-baf5-c6c6f8693d3b">

Site list can then be found by going to settings:

<img width="232" alt="image" src="https://github.com/darkreader/darkreader/assets/6450056/74ee3c3d-7634-4c59-9790-d654a79132f7">

And "site list":

<img width="217" alt="image" src="https://github.com/darkreader/darkreader/assets/6450056/49c54d3e-dca4-43c4-a6d7-680e082d0104">

**Comment by alexanderby** [comment]
There were several bug fixes https://github.com/darkreader/darkreader/commit/a4075ec3f62af642f5c4c2793a6f11484ac5b4e6 and https://github.com/darkreader/darkreader/commit/d3ce1a572e101a545163f444e55058a4065ca5bf

Please reopen if it still happens.
